__tests__/collection.test.ts

```diff
@@ -435,3 +435,34 @@ describe('random thisArg tests', () => {
 		}, array);
 	});
 });
+
+describe('ensure() tests', () => {
+	test('set new value if key does not exist', () => {
+		const coll = new Collection();
+		coll.set('a', 1);
+		coll.set('b', 2);
```

Comment:
```suggestion
		const coll = new Collection([['a', 1], ['b', 2]]);
```

Reply:
Ironically, condensing this to 1 line will set off prettier.  
![image](https://user-images.githubusercontent.com/22574513/143302552-bb4bdbcf-1e5c-41e7-a7ac-68aac024bd5c.png)

I could disable prettier for the 3 locations via 3 comments to force it onto one line.  
![image](https://user-images.githubusercontent.com/22574513/143302714-b311a1d3-7470-4a7b-9bc2-443a4d396993.png)
Or, allow prettier to perform the request change, which results in the original 3 lines becoming 4!
![image](https://user-images.githubusercontent.com/22574513/143302871-d57c5780-7b89-462c-b369-6126b3e72b15.png)

What would you like me to do?

Reply:
I'd definitely not ignore style rules, but let them be.

You can make a function that makes a collection with those entries and returns it, then you do

```js
const coll = createTestCollection();
```

That way we also don't duplicate code.

Reply:
implemented requested changes